tools/windows/DatadogAgentInstaller/CustomActions/UserCustomActions.cs

```diff
@@ -156,6 +166,122 @@ public static ActionResult ReadRegistryProperties(Session session)
             return ReadRegistryProperties(new SessionWrapper(session));
         }
 
+        /// <summary>
+        /// Returns true if we will treat <c>name</c> as an alias for the local machine name.
+        /// </summary>
+        /// <remarks>
+        /// Comparisons are case-insensitive.
+        /// </remarks>
+        private static bool NameIsLocalMachine(string name)
+        {
+            name = name.ToLower();
```

Comment:
would not it be useful to trace this function (local vs not local and even what api call return what (or even if they did or did not fail actually)